src/api/objects/ObjectAPI.js

```diff
@@ -56,11 +58,15 @@ ObjectAPI.prototype.supersecretSetFallbackProvider = function (p) {
  * @private
  */
 ObjectAPI.prototype.getProvider = function (identifier) {
+    //handles the '' vs 'mct' namespace issue
+    const keyString = utils.makeKeyString(identifier);
+    const namespace = this.identifierService.parse(keyString).getSpace();
```

Comment:
This namespace will be set to the defaultSpace (in our case '`mct`') if the namespace is either '' or undefined

Reply:
Done. See: https://github.com/nasa/openmct/pull/3674/files#diff-7b3a993b89a077474a9a4cf7fca79c18131e8e222951d164d2848ab147afbd24R42